src/pycoeus/features.py

```diff
@@ -153,8 +152,7 @@ def extract_flair_features(input_data: ndarray, model_scale=1.0) -> ndarray:
 
     outputs = []
     for i_band in range(n_bands):
-        input_band = normalize_single_band_to_tensor(input_data[i_band:i_band + 1, :, :])[None, :, :, :].float().to(
-            device)
+        input_band = torch.from_numpy(input_data[i_band : i_band + 1, :, :])[None, :, :, :].float().to(device)
```

Comment:
Normalization is removed here. This means that during feature extraction (here) the data is not handled the same way as was done during training of the feature extraction model. The input will therefore likely be out of distribution for the trained model and informative power of the extracted features will be sub-optimal.